Sources/SwiftParser/IncrementalParseTransition.swift

```diff
@@ -345,29 +345,31 @@ public struct ConcurrentEdits: Sendable {
   }
 
   private static func translateSequentialEditsToConcurrentEdits(
-    _ edits: [IncrementalEdit]
-  ) -> [IncrementalEdit] {
-    var concurrentEdits: [IncrementalEdit] = []
+    _ edits: [SourceEdit]
+  ) -> [SourceEdit] {
+    var concurrentEdits: [SourceEdit] = []
     for editToAdd in edits {
       var editToAdd = editToAdd
       var editIndicesMergedWithNewEdit: [Int] = []
       for (index, existingEdit) in concurrentEdits.enumerated() {
-        if existingEdit.replacementRange.intersectsOrTouches(editToAdd.range) {
-          let intersectionLength =
-            existingEdit.replacementRange.intersected(editToAdd.range).length
+        if existingEdit.range.intersectsOrTouches(editToAdd.range),
```

Comment:
I think `range` is not the same as `replacementRange`, so this is a change in behavior, right? 

Same in the line below.